### PR TITLE
ST-243 Guideline to create new projects on Gitlab

### PR DESCRIPTION
**Pull request by adriaan-sac**

### CODE REVIEW

File: src/projects/create_new_project.rst
Comment (flyingfrog81) [inline src/projects/create_new_project.rst]: Add something about the possiblity to add users to the proejct as a result of being maintainer.

File: src/projects/create_new_project.rst
Comment (flyingfrog81) [inline src/projects/create_new_project.rst]: wrap this into a note

File: src/projects/create_new_project.rst
Comment (flyingfrog81) [inline src/projects/create_new_project.rst]: This paragraph must be written more clearly.
There are two sub-groups dedicated to user maintenance.
SKA Reporters is not necessarily for people leaving the organisation (note that when you say SKA Organisation at the moment you are referring to the English company) ... it might bein general for people outside the collaboration.
Remove the reference to how to add a team member, we should add it somewhere else.

File: src/projects/create_new_project.rst
Comment (flyingfrog81) [inline src/projects/create_new_project.rst]: I'd rather say that by default the repository will be created with public access and other setups are possible for private and IP protected repositories upon request.

**Review by flyingfrog81 — CHANGES_REQUESTED** [review]
Good progress, let's iterate one more time accounting for my comments, but I think we are almost there.

**Review by vla22 — COMMENTED** [review]
I think you've addressed Marco's comments well. I'm OK with this being merged. I'll leave it up to Marco to say whether we want to say more about private or IP protected repos - I think given that people have to ask us to create repos at the moment, we can then grill them about why they wish to create non-public repos, so we don't currently need more documentation.

**Review by flyingfrog81 — APPROVED** [review]

**Pull request merged**
